MREUnityRuntime/MREUnityRuntimeLib/Patching/Types/UserPatch.cs

```diff
@@ -15,6 +17,7 @@ public class UserPatch : IPatchable
         public string Name { get; set; }
 
         [PatchProperty]
+        [JsonConverter(typeof(UnsignedConverter))]
```

Comment:
>UnsignedConverter [](start = 30, length = 17)

Is this actually necessary? Newtonsoft can't serialize unsigned ints natively?

Reply:
Newtonsoft can, but `JSON.parse` can't. Javascript doesn't have a concept of unsigned ints, so it stores the data as a signed int. As a result, what goes over the wire is "-1" instead of "4294967295".